repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -41,25 +42,12 @@ final case class LocalProject(
     )
 }
 
-final case class PostProject(name: String)
+final case class PostProject(name: ProjectId)
```

Comment:
This is an incorrect change. Project name is a human-friendly thing (such as "My pet project"), this is not a `projectId` that we control and must be unique.